Remove mentions of `controller..can` choice syntax in 2.8.0 docs (#453)
digital-asset/docs.daml.com: docs/2.8.0/docs/daml/reference/choices.rst

This page gives reference information on choices. For information on the high-level structure of a choice, see :doc:`structure`.

.. _daml-ref-choice-name:

Choice Name
***********

.. literalinclude:: ../code-snippets/Reference.daml
   :language: daml
   :start-after: -- start new choice name snippet

>    :end-before: -- end new choice name snippet

Author: these snippets will likely fail until the daml PR is merged into `main`

.. literalinclude:: ../code-snippets/Reference.daml
   :language: daml
   :start-after: -- start new controller snippet
   :end-before: -- end new controller snippet

- ``controller`` keyword
- The controller is a comma-separated list of values, where each value is either a party or a collection of parties.

  The conjunction of **all** the parties are required to authorize when this choice is exercised.

.. warning::

>   You **must** make sure that the controller parties are observers (or signatories) of the contract, otherwise they won't be able to see the contract (and therefore won't be able to exercise the choice).

Reviewer: ```suggestion
  You **must** make sure that the controller parties are observers (or signatories) of the contract, otherwise they cannot see the contract (and therefore cannot exercise the choice).
```

.. _daml-ref-choice-observers:

Choice Observers
****************

> It is possible for *choice observers* to be attached to a choice using the ``observer`` keyword. The choice observers are a list of parties that, in addition to the stakeholders, will see all consequences of the action.

Reviewer: ```suggestion
*Choice observers* can be attached to a choice using the ``observer`` keyword. The choice observers are a list of parties who are not stakeholders but who see all the consequences of the action.
```
